kuma/wiki/content.py

```diff
@@ -1138,3 +1143,11 @@ def validate_src(self, src):
                     continue
             return True
         return False
+
+
+_content_section_tool = ContentSectionTool()
```

Comment:
Good idea, seems safe from reading the `ContentSectionTool` code (i.e., no residue from a previous `parse` call can pollute a subsequent call).